docs/storage/petalibrary/rclone.md

```diff
@@ -2,12 +2,9 @@
 
 This guide details the process of making a backup for local data on your laptop or lab server to a PetaLibrary allocation. The method employs the software _rclone_, which is a command line application that is available for many architectures. While the following tutorial is tailored for a MacOS user, the general steps to follow are the same and are relevant for Windows and Linux users too.
 
-## Generate public/private keypair for CURC resources
+## Generate public/private keypair for CURC resources (if you wish to not enter a password each time you transfer files)
 
-
-In order to allow for passwordless access to the system, which will enable you to seamlessly back up data, you need to generate a keypair for our system. This is a straightforward process and it is outlined in our [Uploading an SSH Key to CILogon Registry](../../additional-resources/registrycilogon-instructions.md) documentation. Please follow this documentation first before proceeding with the remaining steps. Note that generating a keypair yourself and placing it in `~/.ssh` on our resources will not work, you must follow the provided documentation. 
-
-Once the keypair has been generated, we now need to provide the private key to the client you want to backup data on. Although this next step can't be documented exactly, as every client system will be different, using a secure method (such as scp or sftp) to copy the private key in `~/.ssh/rclone_ssh_key` to the system that you want to back up to PetaLibrary (e.g., your laptop or lab server) is all that is needed.
+In order to allow for passwordless access to CURC, which will enable you to seamlessly back up data, you need to generate a keypair for our system (_note: presently passwordless access is only available to users affiliated with the CU system_). This is a straightforward process and it is outlined in our [Uploading an SSH Key to CILogon Registry](../../additional-resources/registrycilogon-instructions.md) documentation. Please follow this documentation first before proceeding with the remaining steps. Note that generating a keypair yourself and placing it in `~/.ssh` on our resources will not work -- you must follow the documentation noted above. 
 
 **Windows:** Windows users should copy client (e.g. Globus, WinSCP, WSL2, ect) as Admin. Change directories to rclone file location and run:
```

Comment:
This should be made into a "note" admonition (see my comment in this PR for more info)

Reply:
Done